src/utils/work/getExtraDependencies.js

```diff
@@ -17,12 +17,17 @@ const getClusteringSettings = async (experimentId, body, dispatch, getState) =>
   return clusteringSettings;
 };
 
```

Comment:
in this file, line 6: `const getClusteringSettings = async (experimentId, body, dispatch, getState) => {` `body` is highlighted for me as not used. Can you remove it?

Reply:
it is being used